plugins/org.eclipse.n4js/pom.xml

```diff
@@ -123,6 +123,23 @@ Contributors:
 					</executions>
 			</plugin>
 		</plugins>
+		
+		<pluginManagement>
+			<plugins>
+				<!-- TYCHO -->
+				<plugin>
+					<groupId>org.eclipse.tycho</groupId>
+					<artifactId>tycho-compiler-plugin</artifactId>
+					<version>${tycho.version}</version>
+					<configuration>
+						<useProjectSettings>true</useProjectSettings>
+						<compilerArgs>
+							<arg>-nowarn:[${project.basedir}/src-gen${path.separator}${project.basedir}/xtend-gen${path.separator}${project.basedir}/xsemantics-gen]</arg>
+						</compilerArgs>
+					</configuration>
+				</plugin>
+			</plugins>
+		</pluginManagement>
```

Comment:
Actually, this is only a test to disable warnings (which are active right now, I think). This will be copied to other bundle poms later.